internal/action/edit.go

```diff
@@ -97,7 +97,7 @@ func (s *Action) editGetContent(ctx context.Context, name string, create bool) (
 	}
 
 	if !create {
-		out.Yellow(ctx, "Entry %s not found. Creating new secret ...", name)
+		out.Yellow(ctx, "⚠ Entry %s not found. Creating new secret ...", name)
```

Comment:
This could be printed on stderr with a "Warning" prepended using the `out.Warning`, no?

Reply:
Done.